Makefile

```diff
@@ -132,10 +132,16 @@ build/.endnote: build/.mods
 %.endf: %.xml
 	xml2end $< 2>&1 > $@
 
+.PHONY: test
+test:
+	diff -u build/anthology/P19-1007.bib test/data/P19-1007.bib
+	diff -u build/anthology/P19-1007.endf test/data/P19-1007.endf
+	diff -u build/anthology/P19-1007.xml test/data/P19-1007.xml
+
 .PHONY: hugo
 hugo: build/.hugo
 
-build/.hugo: build/.pages build/.bibtex build/.mods build/.endnote
```

Comment:
Please move the dependency on test to a phony target.  In the current version, the phony test target means that the build/.hugo target will have to be built again every time because it depends on a phony target (test) which needs to be built every time.  Add it to site for example

Reply:
Can you look at it now? I think it's correct.

Reply:
Now there is no target which depends on test. I would modify the `site` target as follows:
```
.PHONY: site
site: check bibtex mods endnote hugo split-sitemap test
```
This way it additionally 1) checks whether the XML is valid and 2) perfoms the `test` target once the site is built.

Reply:
Ok, you added it already, didn't show for me for some reason.  Internet is a bit slow, am on a train again.